async-development/js/main.js

```diff
@@ -0,0 +1,35 @@
+const form = document.forms[0];
+const city = form.elements.city;
+const country = form.elements.country;
+
+const weatherTable = document.getElementById('weather-table');
+
+form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+
+    await getWeather(city.value, country.value);
+});
```

Comment:
there is no need to make a callback async
could be replaced with
```suggestion
form.addEventListener('submit', (event) => {
    event.preventDefault();

   getWeather(city.value, country.value);
});
```